dipy/align/vector_fields.pyx

```diff
@@ -1475,16 +1479,39 @@ def simplify_warp_function_3d(floating[:, :, :, :] d,
     r"""
     Simplifies a nonlinear warping function combined with an affine transform
 
-    Modifies the given deformation field by incorporating into it a
-    an affine transformation and voxel-to-space transforms associated to
+    Modifies the given deformation field by incorporating into it
+    an affine transformation and voxel-to-space transforms associated with
     the discretization of its domain and codomain.
+
     The resulting transformation may be regarded as operating on the
     image spaces given by the domain and codomain discretization.
     More precisely, the resulting transform is of the form:
 
     (1) T[i] = W * d[U * i] + V * i
```

Comment:
I think this explanation should not be moved to `Notes` because it is useful to explain the `Parameters` below.